README.md

```diff
@@ -145,11 +145,23 @@ Type: `String[]`<br>
 Default: `[]`
 
 ### timeout
-Timeout for step definitions.
+Timeout in milliseconds for step definitions.
 
 Type: `Number`<br>
 Default: `30000`
 
+### ignoreUndefinedDefinitions
```

Comment:
We could move this into an extra section though, with a short explanation that this is not a vanilla cucumber option, but an option that wdio-cucumber-framework supports.